packages/microservice/src/bus-adapters/externals/mqtt/client-options.ts

```diff
@@ -0,0 +1,176 @@
+import { MqttClient } from './client'
```

Comment:
Does this file come from somewhere? if so, please specify it on the top in order avoid troubles in the future :)